src/main.c

```diff
@@ -2174,6 +2180,114 @@ void on_middle_click() {
     }
 }
 
+/// As a part of [Issue #41]: https://github.com/Team-10-But-Better/Craft/issues/41
+/// To assist the parseConfigToPrintKeybinds() function, this function looks at a one line of the config.h function
+/// and determines if the line is a #define statement defining a Craft keybind.
+int isDefinedCraftKey(char* str, int start, int end)
+{
+    char str2[end - start];
+    for (int i = 0; i < (end - start); i++)
+    {
+        str2[i] = str[start + i];
+    }
+    int tru = 1;
+    if (str2[0] != 'C')
+        tru = 0;
+    if (str2[1] != 'R')
+        tru = 0;
+    if (str2[2] != 'A')
+        tru = 0;
+    if (str2[3] != 'F')
+        tru = 0;
+    if (str2[4] != 'T')
+        tru = 0;
+    return tru;
+}
+
+/// As a part of [Issue #41]: https://github.com/Team-10-But-Better/Craft/issues/41
+/// Since keybinds are created as "#defines" in the config.h file, read all lines of the config.h file
+/// and print all lines of which define Craft keybinds.
+void parseConfigToPrintKeybinds()
```

Comment:
I'm not sure if directly parsing config.h. I think it's generally not considered to be a good idea to look at the code's source files which running. I can't find any better way to do this online though, so perhaps there isn't a better solution.